Lessons/Lesson 3/Solutions/Lesson_3_Solution_JBH.cs

```diff
@@ -0,0 +1,130 @@
+﻿using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Runtime.InteropServices;
+
+double num1 = 0, num2 = 0;  // gloabl variables for basic calculator portion.
+int index = 0, userChoice; // global variables for fibbonacci function.
```

Comment:
you shouldnt have global vars for specific tasks

your structure is fucked, check comment below